owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/GetRemoteCapabilitiesOperation.kt

```diff
@@ -45,211 +50,39 @@ import timber.log.Timber
-                                    respPublic.getBoolean(PROPERTY_MULTIPLE)
-                                )
-                            }
-                        }
-
-                        if (respFilesSharing.has(NODE_USER)) {
-                            val respUser = respFilesSharing.getJSONObject(NODE_USER)
-                            capability.filesSharingUserSendMail = CapabilityBooleanType.fromBooleanValue(
-                                respUser.getBoolean(PROPERTY_SEND_MAIL)
-                            )
-                        }
-
-                        capability.filesSharingResharing = CapabilityBooleanType.fromBooleanValue(
-                            respFilesSharing.getBoolean(PROPERTY_RESHARING)
-                        )
-                        if (respFilesSharing.has(NODE_FEDERATION)) {
-                            val respFederation = respFilesSharing.getJSONObject(NODE_FEDERATION)
-                            capability.filesSharingFederationOutgoing =
-                                CapabilityBooleanType.fromBooleanValue(respFederation.getBoolean(PROPERTY_OUTGOING))
-                            capability.filesSharingFederationIncoming = CapabilityBooleanType.fromBooleanValue(
-                                respFederation.getBoolean(PROPERTY_INCOMING)
-                            )
-                        }
-                        Timber.d("*** Added $NODE_FILES_SHARING")
-                    }
-
-                    if (respCapabilities.has(NODE_FILES)) {
-                        val respFiles = respCapabilities.getJSONObject(NODE_FILES)
-                        // Add files
-                        capability.filesBigFileChunking = CapabilityBooleanType.fromBooleanValue(
-                            respFiles.getBoolean(PROPERTY_BIGFILECHUNKING)
-                        )
-                        if (respFiles.has(PROPERTY_UNDELETE)) {
-                            capability.filesUndelete = CapabilityBooleanType.fromBooleanValue(
-                                respFiles.getBoolean(PROPERTY_UNDELETE)
-                            )
-                        }
-                        if (respFiles.has(PROPERTY_VERSIONING)) {
-                            capability.filesVersioning = CapabilityBooleanType.fromBooleanValue(
-                                respFiles.getBoolean(PROPERTY_VERSIONING)
-                            )
-                        }
-                        Timber.d("*** Added $NODE_FILES")
-                    }
-                }
-                // Result
                 result = RemoteOperationResult(OK)
-                result.data = capability
+                result.data = commonResponse?.ocs?.data?.toRemoteCapability()
 
-                Timber.d("*** Get Capabilities completed ")
+                Timber.d("*** Get Capabilities completed and parsed to ${result.data}")
```

Comment:
Just curious, why these three *** in some logs?

Reply:
Not sure, i just left it the way it was. These three *** are included in other remote operations logs.